packages/@react-spectrum/s2/src/Disclosure.tsx

```diff
@@ -116,7 +116,8 @@ interface DisclosureHeaderProps extends UnsafeStyles, DOMProps {
 
 const headingStyle = style({
   margin: 0,
-  flexGrow: 1
+  flexGrow: 1,
+  display: 'flex'
```

Comment:
if the heading is long, this should probably have flexShrink: 1 and minWidth: 0 so it doesn't break out of flex boundaries